src/renderer/store/Music.js

```diff
@@ -13,12 +13,22 @@ export class Note {
     this.startTime = startTime
     this.duration = duration
     this.pitch = pitch
+    this.id = undefined
```

Comment:
Je comprends pas l'utilité de l'id, il est utilisé où ?

Reply:
@aldebout Dans le `:key` du `v-for` il me semble (c'est ça qui règle le bug au delete).

Reply:
Il est utilisé dans addNote et aussi dans ce qui est suppression de note ! Sans cela on n'arrivait pas à recenser la note pour pouvoir agir sur la box correspondante :)